.travis.yml

```diff
@@ -21,29 +24,45 @@ addons:
       - g++-4.8
       - libxmlsec1-dev
   chrome: stable
+
 env:
   global:
-    - NODE_ENV=production
+    - NODE_ENV=development
```

Comment:
I think we want to keep this at `production` so that our builds will more closely match a production environment.

Reply:
The funny thing is that we were never honoring `NODE_ENV=production`. Any test suites that require yarn deps (e.g. `travis-install-js`) call the `install-yarn` Make recipe, which overrides `NODE_ENV=production`.

Also, you _need_ devDependencies like `jest` to perform the tests themselves - so `NODE_ENV=production` is actually nonsensical, but was overridden by make during `install-yarn`.

Why not just remove this env var then? Because I added `yarn install` to the Travis config. Why not inline with `yarn install` like in the `Makefile`? Because it's better if the base environment variables are all specified in the same place.

Reply:
That sounds good, but when we build our JS assets using webpack, I believe we'll want `NODE_ENV=production`.

Reply:
@billyvg d1b2db7